content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/installation-requirements.md

```diff
@@ -0,0 +1,80 @@
+---
+title : "Chainguard Enforce Agent Installation Requirements"
+description: "Chainguard Enforce Agent Installation Requirements"
+type: "article"
+date: 2023-03-22 13:21:51 +0000 UTC
+draft: false
+images: []
+weight: 799
```

Comment:
Maybe this should be higher up?

Reply:
I pushed it up with network requirements, nice catch.